pkgs/applications/networking/instant-messengers/franz/generic.nix

```diff
@@ -66,12 +76,31 @@ stdenv.mkDerivation rec {
     cups
     expat
     stdenv.cc.cc
-  ];
+  ] ++ lib.optionals enableWayland [ wayland pipewire ];
```

Comment:
Please always build with wayland support if possible

Reply:
ok. I wasn't sure if using wayland shall be always activated.

Reply:
@SuperSandro2000 shall I remove all reference to enableWayland bool ?